### ISSUE TITLE
Adding new journal volumes from the process list and using existing volumes of the chosen (parent) process as a template by the tab "Suche"

### ISSUE THREAD

**Issue by subhhwendt** [kitodo/kitodo-production#4369]
We expect in the tab "Suche" a list of linked volumes for the chosen (parent) process to use as a template for my current volume.

**Comment by subhhwendt** [comment]
Yes - we are looking for the mentioned function from 2.3 to use pattern in process creation for a long list of journal volumes with similiar values. And Arved advised us to look for this feature in the "Search" Tab.

**Comment by matthias-ronge** [comment]
> Does this mean the button should simply be brought back on the process level?

I looked in the code history but found nothing, I think the button never existed there. Yes, it should be added there.

> Or should this only be configurable on the template level and then be applied to _all_ processes created from this template?

No, absolutely not. Since the production templates no longer contain metadata in the current version, the button no longer makes any sense in my opinion for the production templates. Or what do you mean, @subhhwendt?

The property should never be inherited by the processes that are created from the production template; the same applies if the master copy is a process.

> I just realized there _is_ in fact a switch to toggle whether a process should be shown in the selection list, but it can only be set during process creation:

That’s a good thing! @subhhwendt, does that solve your problem?

**Comment by subhhwendt** [comment]
>     I just realized there is in fact a switch to toggle whether a process should be shown in the selection list, but it can only be set during process creation:

That’s a good thing! @subhhwendt, does that solve your problem?

Yes!! But what is the result of switching on "In Auswahlliste anzeigen" at the moment? The connection to "Search" in processcreation is still missing?
And finally it needs a button to switch off this property in the part of "Edit Process" - I guess.

**Comment by matthias-ronge** [comment]
> what is the result of switching on "In Auswahlliste anzeigen" at the moment? The connection to "Search" in processcreation is still missing?

It should be: if you switch “show in selection list” **on**, then the process should appear in the “search” tab when you create a new process, and you can copy over the values, as this function in version 2 was:

![Screenshot](https://user-images.githubusercontent.com/3040657/117659878-c3a02900-b19c-11eb-9d78-323b58a76420.png)

> And finally it needs a button to switch off this property in the part of "Edit Process" - I guess.

Yes, the switch is missing here:

![editprocess](https://user-images.githubusercontent.com/3040657/117785727-251bd280-b245-11eb-96f8-0ade6583143a.png)

It should be possible to switch this on and off for an existing process.

**Comment by henning-gerhardt** [comment]
Only a small suggestion: Please rename "In Auswahlliste anzeigen" into something more understandable. In the past I must delete this flag entry from a lot of processes because users did not know that this flag was only useful if you want to create a new process from a existing process. They thought that this flag is needed to get this process displayed in "my tasks" or after searching for this process or in other lists.

**Comment by solth** [comment]
I would also suggest to rename the `Search` tab in the Import mask - it is a really confusing name and - given the context - could just as well mean "Search in OPAC".

**Comment by solth** [comment]
> It should be: if you switch “show in selection list” **on**, then the process should appear in the “search” tab when you create a new process, and you can copy over the values, as this function in version 2 was:

I just tested this and apparently it _partly_ works:
- switching on "In Auswahlliste anzeigen" during process creation does indeed result in the new process being available in the "Search" tab pull down menu next time a process is created in the same project
- **_but_** the button labeled "Übernehmen" does not actually copy the metadata from the selected process. Apparently the action called in the backend still relies on the metadata being stored in `Property` objects and only copies those, but not actual metadata 

Bottom line: open todos are
- add switch to "Process edit" page to allow toggling its value for existing processes
- fix "Übernehmen" functionality in `SearchTab.java` to copy metadata instead of properties
